lightning/src/onion_message/messenger.rs

```diff
@@ -243,6 +243,46 @@ impl OnionMessageRecipient {
 	}
 }
 
+
+/// The `Responder` struct creates an appropriate [`ResponseInstruction`]
+/// for a response based on our ability to respond.
+pub struct Responder {
+	/// The path along which the response can be sent.
+	reply_path: BlindedPath,
+}
+
+impl Responder {
+	/// Creates a new [`Responder`] instance with the provided reply path.
+	pub fn new(reply_path: BlindedPath) -> Self {
+		Responder {
+		    reply_path,
+		}
+	}
+
+	/// Creates the appropriate [`ResponseInstruction`] for a given response.
+	pub fn respond<T: OnionMessageContents>(self, response: T) -> ResponseInstruction<T> {
+		ResponseInstruction::WithoutReplyPath(OnionMessageResponse {
+			message: response,
+			reply_path: self.reply_path
+		})
+	}
+}
+
+/// This struct contains the information needed to reply to a received message.
+#[allow(unused)]
```

Comment:
Is this needed?

Reply:
The commit was triggering some "unused" warnings (and I believe some failures) in commit-wise CI.
So I added this to prevent those ci issues.